examples/tv-app/android/java/ContentLauncherManager.cpp

```diff
@@ -122,10 +86,10 @@ void ContentLauncherManager::InitializeWithObjects(jobject managerObject)
         env->ExceptionClear();
     }
 
-    mGetSupportedStreamingTypesMethod = env->GetMethodID(ContentLauncherClass, "getSupportedStreamingTypes", "()[I");
-    if (mGetSupportedStreamingTypesMethod == nullptr)
+    mGetSupportedStreamingProtocols = env->GetMethodID(ContentLauncherClass, "getSupportedStreamingProtocols", "()[I");
```

Comment:
You changed the JNI here, did you also changed and tested the related java file?

Reply:
This was work in progress. Now I've added final version.